pkg/agent/nodeportlocal/portcache/port_table_windows.go

```diff
@@ -30,15 +30,6 @@ const (
 	stateInUse protocolSocketState = 1
 )
 
-// podIPPortFormat formats the ip, port to string ip:port.
-func podIPPortProtoFormat(ip string, port int, protocol string) string {
-	return fmt.Sprintf("%s:%d:%s", ip, port, protocol)
-}
-
-func (pt *PortTable) getEntryByPodIPPortProto(ip string, port int, protocol string) *NodePortData {
-	return pt.PodEndpointTable[podIPPortProtoFormat(ip, port, protocol)]
-}
-
 func (pt *PortTable) GetEntry(ip string, port int, protocol string) *NodePortData {
```

Comment:
This function is duplicated for the Linux and Windows implementations I think. I don't a difference between the 2 versions.

Reply:
This function has been moved to port_table.go. So both linux and windows implementation can call it.

Reply:
Have you made this change, I don't see it?